cdci_data_analysis/analysis/parameters.py

```diff
@@ -408,7 +408,7 @@ def check_float_value(value, units=None, name=None):
             try:
                 value = float(value)
             except:
-                raise RuntimeError('type %s not valid for %s' % (type(value), name))
+                raise RuntimeError(f'unable to interpret value {value} (of type {type(value)}) as float')
```

Comment:
funny enough, I had applied this change also in the other PR #280 , before checking this

Reply:
this is the merge on `parameters-default-format`, hence any visible change should not be yet in `parameters-default-format` - in #280 . If it is, rebase before merge will hide it

Reply:
I indeed appeared as a conflict and I resolved it - by picking a slightly longer and more detailed message. Do you have better suggestions for the text?